gcc/testsuite/g++.dg/contracts/cpp26/noexcept-enforce2.C

```diff
@@ -4,11 +4,9 @@
 #include <exception>
 #include <cstdlib>
 
-// Test that there is an active exception when we reach the terminate handler.
 void my_term()
 {
-  try { throw; }
-  catch(int) { std::exit(0); }
+  std::exit(0);
```

Comment:
What is this change supposed to do? I'm testing that a throwing custom violation handler causes terminate called with an exception active, and you're removing that testing here.